ui/kotlinx-coroutines-javafx/build.gradle.kts

```diff
@@ -3,13 +3,20 @@
  */
 
 plugins {
-    id("org.openjfx.javafxplugin")
+    id("org.openjfx.javafxplugin") version "0.0.9"
 }
 
 javafx {
     version = version("javafx")
     modules = listOf("javafx.controls")
-    configuration = "compile"
+    configuration = "compileOnly"
+}
+
+sourceSets {
+    test.configure {
+        compileClasspath += configurations.compileOnly
+        runtimeClasspath += configurations.compileOnly
+    }
```

Comment:
Looking through options provided by the JavaFX plugin, I don't see a better way to do this. Looking at discussions where other projects face the same problem (https://github.com/openjfx/javafx-gradle-plugin/issues/85, https://stackoverflow.com/questions/60249348/run-gradle-tests-using-image-from-badass-runtime-plugin and some others), it seems that the community didn't yet find a clean solution either.